pkgs/by-name/sd/SDL_compat/package.nix

```diff
@@ -66,24 +97,17 @@ stdenv.mkDerivation (finalAttrs: {
   patches = [ ./find-headers.patch ];
   setupHook = ./setup-hook.sh;
 
-  postFixup = ''
-    for lib in $out/lib/*${stdenv.hostPlatform.extensions.sharedLibrary}* ; do
-      if [[ -L "$lib" ]]; then
-        ${
-          if stdenv.hostPlatform.isDarwin then
-            ''
-              install_name_tool ${
-                lib.strings.concatMapStrings (x: " -add_rpath ${lib.makeLibraryPath [ x ]} ") finalAttrs.buildInputs
-              } "$lib"
-            ''
-          else
-            ''
-              patchelf --set-rpath "$(patchelf --print-rpath $lib):${lib.makeLibraryPath finalAttrs.buildInputs}" "$lib"
-            ''
-        }
-      fi
-    done
-  '';
+  passthru.tests = {
+    pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+
+    inherit
+      SDL_image
+      SDL_ttf
+      SDL_mixer
+      SDL_sound
+      tennix
```

Comment:
why tennix specifically? I'd much prefer having `dosbox` here, dosbox -> bottles actually goes into a popular piece of software.